src/components/Toolbar/LinkToolbar.js

```diff
@@ -47,13 +47,15 @@ class LinkToolbar extends React.Component<Props, State> {
     this.originalValue = this.props.link.data.get("href");
     this.setState({ isEditing: !!this.originalValue });
 
-    setImmediate(() =>
-      window.addEventListener("click", this.handleOutsideMouseClick)
-    );
+    typeof window !== "undefined" &&
```

Comment:
Sorry to nit on a relatively small PR, but do you mind changing these to regular `if` statements – I think it's easier for folks to understand, particularly on these multi-line ones. But for consistency, everywhere would be great.

Reply:
Hey, no need to apologize. We also enforce strict contributions guidelines at work, especially on making code consistent with the existing codebase. I'll fix that happily :)